modules/doi/includes/admin.form.inc

```diff
@@ -30,7 +30,48 @@ function islandora_doi_admin_form(array $form, array &$form_state) {
       '#description' => t('An identifier to call yourself, for the OpenURL endpoint. To use this service you first need to register for an account here: http://www.crossref.org/requestaccount/'),
       '#default_value' => variable_get('islandora_doi_openurl_pid', 'user@example.com'),
     ),
+    'save_configuration' => array(
+      '#type' => 'submit',
+      '#name' => 'saveConfiguration',
```

Comment:
Shouldn't need to explicitly set a `#name` for the submit.  Also, `#default_value` is not a valid property on a submit element: https://api.drupal.org/api/drupal/developer!topics!forms_api_reference.html/7.x#submit. May be rendered moot by my comment below.

Reply:
Once again, just copying from https://github.com/Islandora/islandora_usage_stats/blob/7.x/includes/admin.form.inc

Reply:
Should get changed in usage stats as well then.

Reply:
So I just need to remove `#name` and `#default_value` then?

Reply:
`#default_value` becomes `#value`, `#name` can be removed.